content/en/docs/ambient/install/platform-prerequisites/index.md

```diff
@@ -1,73 +1,77 @@
 
-### MicroK8s
+- Be installed into `kube-system` (_not_ `istio-system`)
+- Be installed into another namespace (such as `istio-system`) in which a ResourceQuota has been manually created, for example:
 
-1. If you are using [MicroK8s](https://microk8s.io/), you must append `--set values.cni.cniConfDir=/var/snap/microk8s/current/args/cni-network --set values.cni.cniBinDir=/var/snap/microk8s/current/opt/cni/bin` to the `helm install` command, as MicroK8s [uses nonstandard locations for CNI configuration and binaries](https://microk8s.io/docs/change-cidr).
+{{< text syntax=yaml snip_id=none >}}
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: gcp-critical-pods
+  namespace: istio-system
+spec:
+  hard:
+    pods: 1000
+  scopeSelector:
+    matchExpressions:
+    - operator: In
+      scopeName: PriorityClass
+      values:
+      - system-node-critical
+{{< /text >}}
 
-### K3D
+### k3d
 
-1. If you are using [k3d](https://k3d.io/) with the default flannel CNI, you must append `--set values.cni.cniConfDir=/var/lib/rancher/k3s/agent/etc/cni/net.d --set values.cni.cniBinDir=/bin/` to your `istioctl install` or `helm install` command to install Istio with the `ambient` profile.
+If you are using [k3d](https://k3d.io/) with the default flannel CNI, you must append `--set values.cni.cniConfDir=/var/lib/rancher/k3s/agent/etc/cni/net.d --set values.cni.cniBinDir=/bin/` to your `istioctl install` or `helm install` command to install Istio with the `ambient` profile.
 
-1. Create a cluster and disable `Traefik` so it doesn't conflict with Istio's ingress gateways:
+1. Create a cluster with Traefik disabled so it doesn't conflict with Istio's ingress gateways:
 
     {{< text bash >}}
     $ k3d cluster create --api-port 6550 -p '9080:80@loadbalancer' -p '9443:443@loadbalancer' --agents 2 --k3s-arg '--disable=traefik@server:*'
     {{< /text >}}
 
-1.  Install Istio with the `ambient` profile using `istioctl`:
+1.  Install Istio with the `ambient` profile:
 
     {{< text bash >}}
     $ istioctl install --set profile=ambient --skip-confirmation  --set values.cni.cniConfDir=/var/lib/rancher/k3s/agent/etc/cni/net.d --set values.cni.cniBinDir=/bin
     {{< /text >}}
 
-### K3S
+### K3s
+
+When using [K3s](https://k3s.io/) and one of its bundled CNIs, you must append `--set values.cni.cniConfDir=/var/lib/rancher/k3s/agent/etc/cni/net.d --set values.cni.cniBinDir=/var/lib/rancher/k3s/data/current/bin/` to your `istioctl install` or `helm install` command to install Istio ambient, as K3S uses nonstandard locations for CNI configuration and binaries. These nonstandard locations may be overridden as well [according to K3s documentation](https://docs.k3s.io/cli/server#k3s-server-cli-help). If you are using K3s with a custom, non-bundled CNI, you must use the correct paths for those CNIs, e.g. `/etc/cni/net.d` - [see K3s docs for details](https://docs.k3s.io/networking/basic-network-options#custom-cni).
```

Comment:
Note that because `istioctl` is weird, `--sets` are not cross-compatible between Helm and `istioctl` which will catch copypasters.

- For helm it's `--set cni.cniConfDir=/var/lib/rancher/k3s/agent/etc/cni/net.d`
- For istioctl it's `--set values.cni.cniConfDir=/var/lib/rancher/k3s/agent/etc/cni/net.d`